xbmc/addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.cpp

```diff
@@ -365,6 +418,9 @@ typedef struct EpgEventStateChange
 
 void CAddonCallbacksPVR::UpdateEpgEvent(const EpgEventStateChange &ch, bool bQueued)
 {
+  if (!m_enabled)
+    return;
+
```

Comment:
<code>CAddonCallbacksPVR::UpdateEpgEvent</code> is not a callback function. It is a private member function, only called by <code>CAddonCallbacksPVR::PVREpgEventStateChange</code>. => https://github.com/xbmc/xbmc/blob/master/xbmc/addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.h#L237 So, no need for the enabled check here.